.github/workflows/ci.yaml

```diff
@@ -139,11 +143,22 @@ jobs:
         with:
           version: v0.11.1
           image: kindest/node:v1.21.1
+    
+      - uses: geekyeggo/delete-artifact@v1
+        with:
+          name: docker.tar.gz
+          failOnError: false
+
+      - name: Load images from cache
+        run: |
+          echo "loading docker images..."
+          pigz -dc docker.tar.gz | docker load
```

Comment:
nit: I think `docker load < docker.tar.gz` is enough

Reply:
As I'm migrating from the other CI, I wouldn't want to break this right now (don't know if this is a gh action restriction).

But I'm a +1 of opening a good first issue to investigate and maybe remove this :D

Reply:
OK